Dockerfile

```diff
@@ -16,9 +11,13 @@ RUN apt-get update \
   libv8-dev \
   && apt-get clean
 
-COPY renv.lock .
-#RUN R --quiet -e "renv::restore(repos = c(CRAN = 'https://packagemanager.rstudio.com/all/__linux__/focal/latest'))"
-RUN R --quiet -e "renv::restore(repos = c(CRAN = 'https://cran.rstudio.com'))"
+RUN R -e "install.packages('tidyr')"
```

Comment:
Is there a specific reason we aren't using `renv`?  Without, it we cannot guarantee the container will function the same between each time it is built.  Changes in behavior between versions (especially in dplyr and tidyr) could change things.  This has bitten us more than once in the past.  Could you try to use the rstudio package manager as CRAN repo to get pre-built binaries? `renv::restore(repos = c(CRAN = 'https://packagemanager.rstudio.com/all/__linux__/focal/latest'))`

Reply:
Sure, I can work it back in. I just removed it because it was taking forever to build and it built a lot quicker when I just called out the specific packages. Hopefully a new snapshot will cut down on the packages